platform/src/test/kotlin/uk/gov/gdx/datashare/e2e/AcquirerTest.kt

```diff
@@ -1,9 +1,9 @@
-package uk.gov.gdx.datashare.uk.gov.gdx.datashare.e2e
+package uk.gov.gdx.datashare.e2e
```

Comment:
Ideally, I'd have done this as one change, then the other bits as a later change, this is sort of unrelated to the build layout changes